src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -309,9 +309,6 @@ emit_simd_ins_for_binary_op (MonoCompile *cfg, MonoClass *klass, MonoMethodSigna
 						ins = emit_simd_ins (cfg, klass, OP_XBINOP_BYSCALAR, args [0]->dreg, ins->dreg);
 						ins->inst_c0 = OP_FDIV;
 						return ins;
-					} else if ((fsig->params [0]->type == MONO_TYPE_GENERICINST) && (fsig->params [1]->type == MONO_TYPE_GENERICINST)) {
-						instc0 = OP_FDIV;
-						break;
```

Comment:
I am curious, why was this and similar branch in the multiply case removed? It is disabling SIMD code generation for Vector128<T> div/mul ops for wasm and probably more on the other platforms too. It would help me to find out how to fix it.

@lewing is meanwhile trying to re-enable it in this draft https://github.com/dotnet/runtime/pull/81201 to see if CI would complain.

Reply:
This looks like a merge mistake to me. The correct version should be https://github.com/dotnet/runtime/pull/78784.

Reply:
It looks like it was a bad merge into this PR.  The correct code is in dotnet/runtime@8158cb8be51 (from #78784) but then when the PR was merged an incomplete version from this PR was picked instead during the merge (67abfd0a56b)

Reply:
Sorry for that. I don't remember a merge conflict coming up, so I'd guess this was automatically done on git's end.

Reply:
Cool, thanks for looking into that and confirming it.